source/views/layouts/govuk_template.html.erb

```diff
@@ -12,9 +12,13 @@
     <!--[if IE 8]><link href="<%= asset_path "govuk-template-ie8.css" %>" media="screen" rel="stylesheet" /><![endif]-->
     <link href="<%= asset_path "govuk-template-print.css" %>" media="print" rel="stylesheet" />
 
+    <% if content_for?(:fonts_css) %>
+      <%= yield(:fonts_css) %>
+    <% else %>
```

Comment:
I'm not sure this else is needed since we're providing a default?

Reply:
Yeah, I tried to use the else before I put that variable in the other file, but this didnt work, whereas that did, should have taken it out, have done now.